refinery/ui/source/js/dashboard/partials/primary-group-button.html

```diff
@@ -34,7 +38,8 @@
     popover-placement="right"
     uib-popover-html="'This is your <strong>Primary Group</strong>. Once selected, data
     sets shared with this group will be displayed by default. To change your Primary Group
-    selection, use the drop down menu.'"
+    selection, use the drop down menu. To enable the group buttons, you must
+    be a member of a group.'"
```

Comment:
This is getting long enough that I'm not sure how likely a user is to read the whole thing carefully... This part `To change your Primary Group selection, use the drop down menu.` seems kind of obvious: Perhaps it could be dropped to keep the whole thing more concise?